src/Symfony/Component/HttpFoundation/Session/Storage/Handler/PdoSessionHandler.php

```diff
@@ -529,12 +519,11 @@ private function doRead($sessionId)
             // until other connections to the session are committed.
             try {
                 $insertStmt = $this->pdo->prepare(
-                    "INSERT INTO $this->table ($this->idCol, $this->dataCol, $this->lifetimeCol, $this->timeCol) VALUES (:id, :data, :lifetime, :time)"
+                    "INSERT INTO $this->table ($this->idCol, $this->dataCol, $this->expiryCol) VALUES (:id, :data, :expiry)"
                 );
                 $insertStmt->bindParam(':id', $sessionId, \PDO::PARAM_STR);
                 $insertStmt->bindValue(':data', '', \PDO::PARAM_LOB);
-                $insertStmt->bindValue(':lifetime', 0, \PDO::PARAM_INT);
-                $insertStmt->bindValue(':time', time(), \PDO::PARAM_INT);
+                $insertStmt->bindValue(':expiry', time(), \PDO::PARAM_INT);
```

Comment:
you can just put `0` here since this is only dummy data that will be updated by the write operation later anyway